examples/dump_pcapng_info_pretty.py

```diff
@@ -7,6 +7,7 @@
 # To make sure all packet types are available
 import scapy.all  # noqa
 import scapy.packet
+import six
```

Comment:
Avoid using `six` — we can just assume Python 3 support now.

Reply:
Missing push?

Reply:
Oh sorry, I was marking things resolved as I fixed them locally. I'm new to this aspect of github. I'll push up those changes now.